docs/usage/configuration-options.md

```diff
@@ -263,6 +264,15 @@ Example config:
 }
 ```
 
+Warning: increasing the `branchConcurrentLimit` increases the time it takes for Renovate to complete rebasing a PR branch when it's behind the target branch.
+If you find that Renovate is too slow when rebasing out-of-date branches, decrease the `branchConcurrentLimit`.
+
+If you have too many concurrent branches which rebase themselves each run, Renovate can take a lot of time to rebase.
+Solutions:
+
+- Decrease the concurrent branch limit (note: this won't go and delete any existing, so won't have an effect until you either merge or close existing ones manually)
+- Remove automerge and/or automatic rebasing (set `rebaseWhen` to `conflicted`). However if you have branch protection saying PRs must be up to date then it's not ideal to remove automatic rebasing.
```

Comment:
Not sure if we need this advice or not, I've basically copy/pasted the comment from the maintainer into here. But maybe that's too much information dump?

Maybe we could add this in the FAQ in a section like "Help Renovate rebases are really slow" instead?
Or maybe we don't need this explained at all....

Reply:
it's ok to leave